README.md

```diff
@@ -1,4 +1,6 @@
 [![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/okfn/chat)
+[![goodtables.io](https://goodtables.io/badge/github/okfn/licenses.svg)](https://goodtables.io/github/okfn/licenses)
+
 
 Machine readable list of open (http://opensource.org/,
```

Comment:
I know not introduced in this PR, but just noticing now -- "open" is not accurate. The list includes non-open licenses.

Reply:
Agreed.